testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/component/SampleExternalWebView.kt

```diff
@@ -25,3 +29,18 @@ class SampleWebViewClient(private val miniAppExternalUrlLoader: MiniAppExternalU
     override fun shouldOverrideUrlLoading(view: WebView?, url: String?): Boolean =
         miniAppExternalUrlLoader.shouldOverrideUrlLoading(url ?: "")
 }
+
+class SampleWebChromeClient(val context: Context) : WebChromeClient() {
+
+    override fun onCreateWindow(
+        view: WebView?,
+        isDialog: Boolean,
+        isUserGesture: Boolean,
+        resultMsg: Message?
+    ): Boolean {
+        // redirect to default browser when there is window.open / _blank type anchor
+        (resultMsg?.obj as WebView.WebViewTransport).webView = WebView(context)
```

Comment:
Maybe we should also check `isUserGesture`? The docs say this can prevent unwanted pop-ups:
```
if (isUserGesture) {
 // ...
} else {
  return false
}
```

What do you think?

Reply:
Yes, it's okay to check with `isUserGesture`, I have updated this part.